homeassistant/components/sleepiq/__init__.py

```diff
@@ -85,6 +107,54 @@ def update(self):
 
         self.beds = {bed.bed_id: bed for bed in beds}
 
+    def set_new_sleep_number(self, bed_name: str, side: str, sleep_number):
+        """Change the sleep number on a side to a new value."""
+
+        # Sanity check: Sleep number has to be between 1-100.
+        if 0 < sleep_number <= 100:
+            self._set_sleep_number(bed_name, side, int(sleep_number))
+        else:
+            message = f"Invalid sleep number: {sleep_number}"
```

Comment:
Please don't format the message before passing it to the logger. Use old style string formatting in combination with the logger to avoid not needed formatting if the logging isn't activated.